wagtail/admin/edit_handlers.py

```diff
@@ -947,6 +954,16 @@ def get_edit_handler(cls):
 Page.get_edit_handler = get_edit_handler
 
 
+@receiver(setting_changed)
+def reset_cache(**kwargs):
```

Comment:
Could you give this a more descriptive name?

Reply:
Will do!